cpp/modmesh/buffer/BufferExpander.hpp

```diff
@@ -127,6 +131,16 @@ class BufferExpander
         return data(it);
     }
 
+    using iterator = int8_t *;
+    using const_iterator = int8_t const *;
+
+    iterator begin() noexcept { return m_begin; }
```

Comment:
@yungyuc I think it's ready to merge.

I think a test for testing the iterator is trivial, what do you think?

Reply:
> @yungyuc I think it's ready to merge.
> 
> I think a test for testing the iterator is trivial, what do you think?

Yes it can be trivial but should be added.